x/stakeibc/keeper/hooks.go

```diff
@@ -146,9 +146,9 @@ func (k Keeper) UpdateRedemptionRates(ctx sdk.Context, depositRecords []recordst
 	// Update the redemption rate for each host zone
 	for _, hostZone := range k.GetAllHostZone(ctx) {
 
-		// Get the redemption rate components
-		stSupply := k.bankKeeper.GetSupply(ctx, types.StAssetDenomFromHostZoneDenom(hostZone.HostDenom)).Amount.Int64()
-		if stSupply == 0 {
+		// Gather redemption rate components
+		stSupply := k.bankKeeper.GetSupply(ctx, types.StAssetDenomFromHostZoneDenom(hostZone.HostDenom)).Amount
+		if stSupply.Equal(sdk.ZeroInt()) {
```

Comment:
nit: Can you just use `.IsZero()` for these checks?

Reply:
done